src/components/Modal/VotingPowerDelegationDetail/VotingPowerDelegationDetail.tsx

```diff
@@ -42,50 +50,98 @@ function VotingPowerDelegationDetail({ delegate, onBackClick }: VotingPowerDeleg
   const [land] = useLandBalance(address, ChainId.ETHEREUM_MAINNET)
   const [ens] = useEnsBalance(address, ChainId.ETHEREUM_MAINNET)
   const [votes] = useAsyncMemo(async () => Snapshot.get().getAddressVotes(SNAPSHOT_SPACE, address), [])
+  const { profile } = useProfile(address)
+  const [isExpanded, setIsExpanded] = useState(false)
+  const [showFadeout, setShowFadeout] = useState('')
+
+  useEffect(() => {
+    if (!isExpanded) {
+      setTimeout(() => {
+        setShowFadeout('')
+      }, 500)
+    } else {
+      setShowFadeout('Fadeout--hidden')
+    }
+  }, [isExpanded])
 
   const mana = mainnetMana + maticMana + (wMana || 0)
   const totalVotingPower = votingPower - delegatedVotingPower
 
   return (
     <>
       <Modal.Header className="VotingPowerDelegationDetail__Header">
-        <Button basic aria-label={t('modal.vp_delegation_detail.backButtonLabel')} onClick={onBackClick}>
+        <Button basic aria-label={t('modal.vp_delegation.backButtonLabel')} onClick={onBackClick}>
           <ChevronLeft />
         </Button>
-        <span>{address}</span>
+        <Avatar size="small" address={candidate.address} />
```

Comment:
The `Avatar` component considers that case, here is an example with an address that was never used in dcl.

![image](https://user-images.githubusercontent.com/45410089/166240267-20ef64b3-a735-409b-8757-751e09fba217.png)

About reusing the component, maybe it could be passed by parameter since the same is requested in the list above. WDYT @andyesp ?